.travis.yml

```diff
@@ -3,11 +3,29 @@ language: python
 python:
   - 2.7
 install:
-  - pip install ansible-lint
-  - rvm install 2.2.4
+  - changes=$(git diff origin/master --name-only | grep "lib\|test" || true)
+  - |
+    if [[ ${changes[@]} ]];then
```

Comment:
Does this test the length? Wouldn't ```grep -q``` and the exit code work just as well?

Reply:
It does. I was not able to get the latter concept to work within the confines of Travis and it's signal checking.

Reply:
I wonder if ```if git diff origin/master --name-only | grep-q "lib\|test" ; then``` would work but if this works then so be it.